src/app_service/service/message/service.nim

```diff
@@ -666,7 +666,7 @@ QtObject:
   proc getMessageByMessageId*(self: Service, messageId: string): GetMessageResult =
     try:
       result = GetMessageResult()
-      let msgResponse = status_go.getMessageByMessageId(messageId)
+      let msgResponse = status_go.preparedMessageByID(messageId)
```

Comment:
Since we seem to only use `preparedMessageByID`, I wonder if we could just improve `getMessageByMessageId` instead of adding the new API.

We can just make `getMessageByMessageId` smarter and have it call `prepareMessage` when it is a "media" message